pylint/checkers/base.py

```diff
@@ -1474,6 +1479,19 @@ def visit_dict(self, node: nodes.Dict) -> None:
                 self.add_message("duplicate-key", node=node, args=key)
             keys.add(key)
 
+    @utils.check_messages("duplicate-value")
+    def visit_set(self, node: nodes.Set) -> None:
+        """Check duplicate value in set."""
+        values = set()
+        for v in node.elts:
+            if isinstance(v, nodes.Const):
+                value = v.value
+            else:
+                continue
+            if value in values:
+                self.add_message("duplicate-value", node=node, args=value)
```

Comment:
```suggestion
                self.add_message("duplicate-value", node=node, args=value, confidence=HIGH)
```

You'll probably need to important that.